code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -134,10 +134,6 @@
 		src.druggy--
 		src.druggy = max(0, src.druggy)
 
-	//update the state of modules and components here
```

Comment:
being unconscious already prevents usage of tools anyways, force-unequipping them was a pointless nuisance, and it interfered with the selfstun from firing hatton